### PR TITLE
docs(app): Improve docs examples with code group to avoid confusion

### PR DESCRIPTION
**Pull request by JaimeTorrealba**

### CODE REVIEW

File: docs/guide/loaders/use-fbx.md [hunk 1]
```diff
@@ -8,4 +8,43 @@ A composable that allows you to easily load glTF models into your **TresJS** sce
 
 ## Usage
 
-<<< @/.vitepress/theme/components/UseFBXDemo.vue{3,7,14-19}
+::: code-group
+```vue [app.vue]
+<script setup lang="ts">
+import { TresCanvas } from '@tresjs/core'
+import { OrbitControls } from '@tresjs/cientos'
+import AsyncComponent from './AsyncComponent.vue'
```
Comment (alvarosabu) [inline docs/guide/loaders/use-fbx.md:16]:
```suggestion
import TheModel from './TheModel.vue'
```

File: docs/guide/loaders/use-fbx.md [hunk 2]
```diff
@@ -8,4 +8,43 @@ A composable that allows you to easily load glTF models into your **TresJS** sce
...
+</script>
+
+<template>
+  <TresCanvas clear-color="#1F90FF">
+    <TresPerspectiveCamera :position="[11, 11, 11]" />
+    <OrbitControls />
+    <Suspense>
+      <AsyncComponent />
```
Comment (alvarosabu) [inline docs/guide/loaders/use-fbx.md:24]:
```suggestion
      <TheModel />
```

File: docs/guide/loaders/use-fbx.md [hunk 3]
```diff
@@ -8,4 +8,43 @@ A composable that allows you to easily load glTF models into your **TresJS** sce
...
+    </Suspense>
+    <TresDirectionalLight
+      :intensity="2"
+      :position="[3, 3, 3]"
+    />
+    <TresAmbientLight :intensity="1" />
+  </TresCanvas>
+</template>
+```
+```vue{2,6} [AsyncComponent.vue]
```
Comment (alvarosabu) [inline docs/guide/loaders/use-fbx.md:34]:
```suggestion
```vue{2,6} [TheModel.vue]
```

File: docs/guide/loaders/use-gltf.md [hunk 4]
```diff
@@ -8,7 +8,43 @@ A composable that allows you to easily load glTF models into your **TresJS** sce
 
 ## Usage
 
-<<< @/.vitepress/theme/components/UseGLTFDemo.vue{3,7,14-16}
+::: code-group
+```vue [app.vue]
+<script setup lang="ts">
+import { TresCanvas } from '@tresjs/core'
+import { OrbitControls } from '@tresjs/cientos'
+import AsyncComponent from './AsyncComponent.vue'
```
Comment (alvarosabu) [inline docs/guide/loaders/use-gltf.md:16]:
```suggestion
import TheModel from './TheModel.vue'
```

File: docs/guide/loaders/use-gltf.md [hunk 5]
```diff
@@ -8,7 +8,43 @@ A composable that allows you to easily load glTF models into your **TresJS** sce
...
+</script>
+
+<template>
+  <TresCanvas clear-color="#F78B3D">
+    <TresPerspectiveCamera :position="[3, 2, 5]" />
+    <OrbitControls />
+    <Suspense>
+      <AsyncComponent />
```
Comment (alvarosabu) [inline docs/guide/loaders/use-gltf.md:24]:
```suggestion
      <TheModel />
```

File: docs/guide/loaders/use-gltf.md [hunk 6]
```diff
@@ -8,7 +8,43 @@ A composable that allows you to easily load glTF models into your **TresJS** sce
...
+    </Suspense>
+    <TresDirectionalLight
+      :intensity="2"
+      :position="[3, 3, 3]"
+    />
+    <TresAmbientLight :intensity="1" />
+  </TresCanvas>
+</template>
+```
+```vue{2,6} [AsyncComponent.vue]
```
Comment (alvarosabu) [inline docs/guide/loaders/use-gltf.md:34]:
```suggestion
```vue{2,6} [TheModel.vue]
```

File: docs/guide/loaders/use-video-texture.md [hunk 7]
```diff
@@ -10,7 +10,47 @@ This composable is based on the Drei [useVideoTexture](https://github.com/pmndrs
 
 ## Usage
 
-<<< @/.vitepress/theme/components/VideoTextureDemo.vue{4,8-9,20}
+::: code-group
+```vue [app.vue]
+<script setup lang="ts">
+import { TresCanvas } from '@tresjs/core'
+import { OrbitControls } from '@tresjs/cientos'
+import AsyncComponent from './AsyncComponent.vue'
```
Comment (alvarosabu) [inline docs/guide/loaders/use-video-texture.md:18]:
```suggestion
import TheVideoTexture from './TheVideoTexture.vue'
```

File: docs/guide/loaders/use-video-texture.md [hunk 8]
```diff
@@ -10,7 +10,47 @@ This composable is based on the Drei [useVideoTexture](https://github.com/pmndrs
...
+
+<template>
+  <TresCanvas clear-color="#333">
+    <TresPerspectiveCamera
+      :position="[0, 5, 9]"
+      :look-at="[0, 1, 0]"
+    />
+    <OrbitControls />
+    <Suspense>
+      <AsyncComponent />
```
Comment (alvarosabu) [inline docs/guide/loaders/use-video-texture.md:29]:
```suggestion
      <TheVideoTexture />
```

File: docs/guide/loaders/use-video-texture.md [hunk 9]
```diff
@@ -10,7 +10,47 @@ This composable is based on the Drei [useVideoTexture](https://github.com/pmndrs
...
+    </Suspense>
+    <TresGridHelper />
+    <TresAmbientLight />
+  </TresCanvas>
+</template>
+```
+```vue{3,8,13} [AsyncComponent.vue]
```
Comment (alvarosabu) [inline docs/guide/loaders/use-video-texture.md:36]:
```suggestion
```vue{3,8,13} [TheVideoTexture.vue]
```

**Review by alvarosabu — CHANGES_REQUESTED** [review]
Great idea @JaimeTorrealba, hopefully this will educate community about async usage on cientos.

I just added minor suggestion on the naming of components. 

Thanks for the PR

**Comment by JaimeTorrealba** [comment]
@alvarosabu I have fixed this one, please let me know if everything good

**Pull request merged**